File: Examples/PHP/Files/DeleteFileFromThirdPartyStorage.php
//ExStart:
<?php
require_once realpath(__DIR__ . '/..') . '/vendor/autoload.php';
require_once realpath(__DIR__ . '/..') . '/Utils.php';

use Aspose\Storage\StorageApi;
use Aspose\Storage\AsposeApp;

class Files {

	public $storageApi;

	public function __construct() {
		AsposeApp::$appSID = Utils::appSID;
		AsposeApp::$apiKey = Utils::apiKey;
		$this->storageApi = new StorageApi();
	}

	public function deleteFile() {
		// Upload file to Aspose Cloud Storage
		$fileName = "testfile.txt";
		Utils::uploadFile($fileName);

        $result = $this->storageApi->DeleteFile($Path=$fileName, $versionId = null, $storage = "MyDropboxStorage");
        print_r($result);
	}
}

$files = new Files();
$files->deleteFile();

?>
//ExEnd:

File: Examples/Ruby/Files/delete_file_from_third_party_storage.rb
#ExStart:
require 'aspose_storage_cloud'

class Files
    
  include AsposeStorageCloud

  def initialize
    # Get App key and App SID from https://cloud.aspose.com
    AsposeApp.app_key_and_sid("App Key", "App SID")
    @storage_api = StorageApi.new
  end

  def upload_file(file_name)
    @storage_api = StorageApi.new 
    response = @storage_api.put_create(file_name, File.open("../../../data/" << file_name,"r") { |io| io.read }, {storage: "MyDropboxStorage"})
  end
  
  def delete_file
    # Upload file to Aspose Cloud Storage
    file_name = "testfile.txt"
    upload_file(file_name)
    
    response = @storage_api.delete_file(file_name, {storage: "MyDropboxStorage"})
  end

end

files = Files.new()
puts files.delete_file
#ExEnd: